app/views/components/input_date.scala.html

```diff
@@ -41,6 +41,18 @@
     seqIDs.mkString(" ")
 }
 
+@errorMessageAndId = @{
+    if (field("day").errors.nonEmpty || field("month").errors.nonEmpty || field("year").errors.nonEmpty) {
+        Seq(Tuple2("error-message-date", messages("messages__error__date")))
+    } else if (field.errors.nonEmpty) {
+        field.errors.map { error =>
+            Tuple2(s"error-message-${field.id}-input", messages(error.message, error.args: _*))
+        }
+    } else {
```

Comment:
I think this is unnecessary. Mapping an empty seq will just give another empty seq

Reply:
remove unnecessary else